modules/alibaba-ai-installing.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_alibaba/installing-alibaba-assisted-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
```

Comment:
This isn't really a procedure as we'd use one. The steps are rather general. A process outline makes more sense either at the assembly level or as a rewritten concept.

Reply:
I had it at the assembly level, but it looked like an overview with more to come. Plus there is just one subsection not related to anything and it looked weird. I'm not sure what you mean by rewritten concept, though I have changed the :_mod-docs-content-type:  to CONCEPT.